<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_data">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>541</width>
      <height>271</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hex</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ASCII</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="button_read">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_write">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
